src/sonic-framework/rebootbackend/rebootbe.cpp

```diff
@@ -47,6 +48,14 @@ void RebootBE::Start() {
   s.addSelectable(&m_Done);
   s.addSelectable(&m_RebootThreadFinished);
 
+
+  if (swss::WarmStart::isWarmStart()) {
+    SWSS_LOG_NOTICE("Launching init thread for warm start");
```

Comment:
There is no init thread, these messages are not needed.

Can you check the implication of SetCurrentStatus to WARM_INIT_WAIT without the init thread ?

Reply:
Kishan, I have removed the message. WARM_INIT_WAIT is required.

Reply:
Have you pushed the changes ? I still see the init thread message (LOG_NOTICE)

Reply:
Kishan, I have pushed the changes now. Can you pl. check.